xchange-examples/src/main/java/org/knowm/xchange/examples/kraken/account/KrakenAccountDemo.java

```diff
@@ -30,11 +36,22 @@ private static void generic(Exchange krakenExchange) throws IOException {
 
     AccountInfo accountInfo = krakenExchange.getAccountService().getAccountInfo();
     System.out.println("Account Info: " + accountInfo.toString());
+
+    fundsInfo(krakenExchange.getAccountService());
   }
 
   private static void raw(Exchange krakenExchange) throws IOException {
 
     KrakenAccountServiceRaw rawKrakenAcctService = (KrakenAccountServiceRaw) krakenExchange.getAccountService();
     System.out.println("Balance Info: " + rawKrakenAcctService.getKrakenBalance());
   }
+
+  private static void fundsInfo(AccountService accountService) throws IOException {
```

Comment:
This should be made generic, i.e. no references to Kraken. See [this comment](https://github.com/timmolter/XChange/pull/1406#issuecomment-288047134).